apps/mobile/src/components/Feed/CommentsBottomSheet/CommentsBottomSheetLine.tsx

```diff
@@ -99,21 +125,47 @@ export function CommentsBottomSheetLine({
           </GalleryTouchableOpacity>
         </View>
       )}
-      <View className="flex flex-col grow-0">
-        <View className="flex flex-row space-x-1">
-          <Typography className="text-sm leading-4" font={{ family: 'ABCDiatype', weight: 'Bold' }}>
-            {comment.commenter?.username}
-          </Typography>
-          <Typography
-            className="text-xxs text-metal leading-4"
-            font={{ family: 'ABCDiatype', weight: 'Regular' }}
+      <View className="flex flex-col w-full flex-1">
+        <View className="flex-row justify-between">
+          <View>
+            <View className="flex-row space-x-1">
+              <Typography
+                className="text-sm leading-4"
+                font={{ family: 'ABCDiatype', weight: 'Bold' }}
+              >
+                {comment.commenter?.username}
+              </Typography>
+              <Typography
+                className="text-xxs text-metal leading-4"
+                font={{ family: 'ABCDiatype', weight: 'Regular' }}
+              >
+                {timeAgo}
+              </Typography>
+            </View>
+            <ProcessedText text={comment.comment} mentionsRef={nonNullMentions} />
+          </View>
+          <GalleryTouchableOpacity
+            className="flex-row items-center gap-0.5"
+            onPress={handleAdmirePress}
+            eventElementId="Press Admire Comment"
```

Comment:
nit: the `eventElementId` should be the name of the element itself so we don't need to add a verb ("press"), this can be something like "Admire Comment" or "Admire Comment Button"

Reply:
Updated!